app/screens/Login.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+  Text,
+  View,
+  TextInput,
+  Button,
+  StyleSheet,
+  TouchableOpacity,
+} from "react-native";
+import {
+  FirebaseRecaptchaVerifierModal,
+  FirebaseRecaptchaBanner,
+} from "expo-firebase-recaptcha";
+import useFirebase from "../hooks/useFirebase";
+import { app } from "../../firebase/firebase";
+export default function LoginScreen() {
+
+  const { options } = app;
+  const attemptInvisibleVerification = false;
+  const authByPhone = useFirebase();
+
+  return (
+    <View style={{ padding: 20, marginTop: 50 }}>
```

Comment:
[nit picks] Another personal recommendation, move in-line styles to `StyleSheet.create`

Reply:
can you tell us where is the right place to put firebase configuration, please?

Reply:
for the styles, I would keep them in the same file.

Reply:
take a look at the first example [here](https://reactnative.dev/docs/0.66/style)

Reply:
Hi Mazen, Thank you  for your support, even if I want configuration in another files?

Reply:
@Nizar7zak not sure what you mean. Are you talking about the `userFirebase` hook or styles?

I was just mentioning to move this files styles to the `StyleSheet.create` function in the same file. I usually tend to have styles per file and not reusable styles as that can get overly complicated.

If easier, here is my Discord: mazenchami#9806. We can chat there or even voice call

Reply:
Yes I asked about useFireBase hook?

Reply:
@Nizar7zak regarding `useFirebase` hook, I think you have everything in the right way. I would maybe the folder `/firebase` to `/app/firebase` and keep everything in your `app` folder